### ISSUE TITLE
`unowned(unsafe)` transformed to `unowned(_)` in closures

### ISSUE THREAD

**Issue by b8591340** [nicklockwood/SwiftFormat#967]

**Comment by nicklockwood** [comment]
@b8591340 thanks for reporting this - can you provide the surrounding code so I can understand the context?

**Comment by b8591340** [comment]
Hi Nick! Thanks for SwiftFormat! Here’s a minimal repro.

```swift
import Combine

class Bug {
    var num = 0
    func method() {
        Just(1)
            .sink { [unowned(unsafe) self] in
                num += $0
            }
    }
}
```

**Comment by nicklockwood** [comment]
@b8591340 fixed in 0.48.7
